### DISCUSSION TITLE
Atlas bibliocraft hack

### DISCUSSION THREAD

**Discussion by Adamvvs100** [community/community#118284] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

FRANÇAIS :
Bonjour, est ce que quelqu'un pourrait me dire où installer le mod pour se donner des items avec un atlas sur bibliocraft, merci a vous

ENGLISH : 
Hello, could someone tell me where to install the mod to give items with an atlas on bibliocraft, thank you

**Comment by Aiakos13** [comment]
response #1

**Comment by ebndev** [comment]
Hi [Adamvvs100](https://github.com/Adamvvs100),

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on your issue. The GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
